packages/ui/src/hooks/useQrCode.ts

```diff
@@ -0,0 +1,9 @@
+import { useMeasure } from 'react-use';
+
+export default function useQrCode() {
```

Comment:
Let's rename this hook to it's more clear about its purpose.
```suggestion
export default function useQrCodeSize() {
```